### DISCUSSION TITLE
Could you point me to examples of this package showing line-by-line coverage in a GitHub Pull Request?

### DISCUSSION THREAD

**Discussion by shafiquejamal** [ArtiomTr/jest-coverage-report-action#261] [Q&A] [answered]
Hello all, 

Could anyone point me to examples of what the line-by-line coverage in a GitHub pull request would look like if I use this package? I'm looking for something like this, which is from [Atlassian](https://www.atlassian.com/continuous-delivery/software-testing/code-coverage):

<img width="799" alt="Screen Shot 2022-04-18 at 2 35 11 PM" src="https://user-images.githubusercontent.com/2116061/163896418-a49e501e-b689-44a2-a6b0-51de60bc16df.png">

Or this, which is from [Codecov](https://about.codecov.io/product/feature/source-code-coverage/):

<img width="890" alt="Screen Shot 2022-04-18 at 8 21 18 PM" src="https://user-images.githubusercontent.com/2116061/163896894-5015f531-4aa1-4529-9aca-fc7430a1aeaf.png">

The `README.md` for this package `ArtiomTr/jest-coverage-report-action` states that one of the features is "Failed tests & uncovered line annotations", but I don't see coverage in any of the open PRs.

How can I see examples of the line-by-line coverage annotations that this package provides? Thanks!

**Comment by ArtiomTr** [comment, marked as answer]
Hello @shafiquejamal :wave:,

Annotation options are documented [here](https://github.com/ArtiomTr/jest-coverage-report-action#change-annotations). Also, please note that you won't get the result, that Atlassian or Codecov has - GitHub UI doesn't allow any modifications, so the action only adds **annotations**, which look like this:
![image](https://user-images.githubusercontent.com/44021713/164063006-a8e0d574-775a-47c2-be80-5bd9a6dc77cb.png)
